pkg/cli/cmds/server.go

```diff
@@ -603,6 +604,12 @@ var ServerFlags = []cli.Flag{
 		Usage:       "(experimental) Run rootless",
 		Destination: &ServerConfig.Rootless,
 	},
+	&cli.StringFlag{
+		Name:        "secrets-encryption-key-type",
+		Usage:       "(experimental) Secret encryption key type (valid values: 'aescbc', 'secretbox')",
+		Destination: &ServerConfig.EncryptKeyType,
+		Value:       "aescbc",
+	},
```

Comment:
Nit: these are documented upstream as "encryption providers", not key types. Ref: https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/#providers
```suggestion
	&cli.StringFlag{
		Name:        "secrets-encryption-provider",
		Usage:       "(experimental) Secret encryption provider (valid values: 'aescbc', 'secretbox')",
		Destination: &ServerConfig.EncryptProviderName,
		Value:       "aescbc",
	},
```